driver/SHiELD/atmosphere.F90

```diff
@@ -672,15 +721,6 @@ subroutine atmosphere_resolution (i_size, j_size, global)
 
  end subroutine atmosphere_resolution
 
-
```

Comment:
I am curious as to why these routines needed to be moved to an include file instead of being in `atmosphere.F90` unless it is to prevent that file from becoming too long. Also are these routines the same as in the other drivers? If so, then we might want to move them to a common file instead of having to maintain these routines for all of the different drivers.

Reply:
It allows you to write the basic logic once and then create both 32- and 64-bit versions via macro definitions to have capability of choosing the desired precision at runtime based on the argument type.

Reply:
Cool. Thank you for your explanation.